config/objects/generic.json

```diff
@@ -72,12 +76,15 @@
 					"zoneLimit"	: 1,
 					"value"		: 100,
 					"rarity"	: 100
-				}
+				},
+				"modes" : ["resource-resource", "resource-player"],
+				"efficiency" : 5,
+				"title" : "core.genrltxt.159"
 			}
 		}
 	},
 	"freelancersGuild" : {
-		"index" :213,
+		"index" :213, //deprecated
```

Comment:
Not really deprecated since we need to load this object from h3m

Reply:
That’s why I leave it

Reply:
I meant - don't see any need to mark it as deprecated since this is part of h3m loader

Reply:
disagree, but ok can remove

Reply:
to point out - I was thinking to have it as hint for modders that they shouldn't use index fields for markets

Reply:
That should not be a problem - "index" fields set in mods other than "core" should trigger validation warning and likely lead to a crash (since such object would conflict with our object).

So mods can't really use index field even if they attempt to